en/docs/learn/consume-api/manage-application/generate-keys/obtain-access-token/changing-the-default-token-expiration-time.md

```diff
@@ -2,40 +2,45 @@
 
 Follow the instructions below to change the default token expiry time based on your requirements.
 
+## Changing the default token expiration time at the global-level
 
-## Changing the default expiration time of application access tokens
+!!! note
```

Comment:
@tharindu1st , is the placement of this note correct? As in, is this applicable for global changes or application level changes or both?